script/pageload/functions.js

```diff
@@ -16,7 +16,7 @@ var insertAfter = function(newNode, referenceNode) {
 getAllBounties = function(callback){
   var bounties_api_url = 'https://gitcoin.co/api/v0.1/bounties/?idx_status=open&network=mainnet&order_by=-web3_created';
   var xmlHttp = new XMLHttpRequest();
-  xmlHttp.open( "GET", bounties_api_url, true ); // false for asynchronous request
+  xmlHttp.open( "GET", bounties_api_url, true ); // true for asynchronous request
```

Comment:
Thinking more about it, we can also omit the third argument as it defaults to `true`. 

What do you guys think of removing the third argument and the comment as well?